docs/usage/shoot_high_availability.md

```diff
@@ -49,4 +49,16 @@ If you already have a shoot cluster with non-HA control plane then following upg
 
 **Disallowed Transitions**
 
-If you have already set-up a HA shoot control plane with `node` failure tolerance then an upgrade to `zone` failure tolerance is currently not supported, mainly because already existing volumes are bound to the zone they were created in.
\ No newline at end of file
+If you have already set-up a HA shoot control plane with `node` failure tolerance then an upgrade to `zone` failure tolerance is currently not supported, mainly because already existing volumes are bound to the zone they were created in.
+
+## Zone Outage Situation
+
+An availability zone outage might lead to the requirement to change your cluster setup temporarily and on short notice, in order to compensate failures and shortages resulting from the outage.
+For instance, if the shoot cluster has worker nodes across three zones where one zone goes down, the computing power from these nodes is also gone during that time.
+Changing the worker pool (`shoot.spec.provider.workers[]`) and infrastructure (`shoot.spec.provider.infrastructureConfig`) configuration can eliminate this disbalance, having enough machines in healthy availability zones that can cope with the requests of your applications.
+
+Gardener relies on a sophisticated reconciliation flow with several dependencies for which various flow steps wait for the _readiness_ of prior ones.
+During a zone outage this can block the entire flow, e.g. because all three `etcd` replicas can never be ready when a zone is down, and required changes mentioned above can never be accomplished.
```

Comment:
```suggestion
During a zone outage, this can block the entire flow, e.g. because all three `etcd` replicas can never be ready when a zone is down, and required changes mentioned above can never be accomplished.
```